packages/spectral/src/types/IntegrationDefinition.ts

```diff
@@ -77,65 +98,80 @@ export type FlowExecutionContext = ActionContext<
+  /**
+   * Optional list of API key(s) to use for the endpoint of this flow
+   * when the endpoint security type is `EndpointSecurityType.Organization`.
+   */
   organizationApiKeys?: string[];
-  /** Optional schedule configuration that defines the frequency with which this Flow will be automatically executed. */
+  /** Optional schedule configuration that defines the frequency with which this flow will be automatically executed. */
   schedule?: (ValueExpression<string> | ConfigVarExpression) & {
     timezone?: string;
   };
   /** Optional error handling configuration. */
   errorConfig?: StepErrorConfig;
-  /** Specifies the trigger function for this Flow, which returns a payload and optional HTTP response. */
+  /** Specifies the trigger function for this flow, which returns a payload and optional HTTP response. */
   onTrigger?:
     | TriggerReference
     | TriggerPerformFunction<Inputs, ConfigVars, false, TriggerResult<false, TTriggerPayload>>;
-  /** Specifies the function to execute when an Instance of this Integration is deployed. */
+  /**
+   * Specifies the function to execute when an instance of this integration is deployed. See
+   * https://prismatic.io/docs/custom-connectors/triggers/#instance-deploy-and-delete-events-for-triggers
+   */
   onInstanceDeploy?: TriggerEventFunction<Inputs, ConfigVars>;
-  /** Specifies the function to execute when an Instance of an Integration is deleted. */
+  /**
+   * Specifies the function to execute when an instance of an integration is deleted. See
+   * https://prismatic.io/docs/custom-connectors/triggers/#instance-deploy-and-delete-events-for-triggers
+   */
   onInstanceDelete?: TriggerEventFunction<Inputs, ConfigVars>;
-  /** Specifies the main function for this Flow */
+  /** Specifies the main function for this flow which is run when this flow is invoked. */
   onExecution: FlowOnExecution<TTriggerPayload>;
 }
 
-/** Defines attributes of a Preprocess Flow Configuration used by a Flow of an Integration. */
+/** Defines attributes of a Preprocess flow Configuration used by a flow of an integration. */
 export type PreprocessFlowConfig = {
-  /** Name of the field in the data payload returned by the Preprocess Flow to use for a Flow Name. */
+  /** Name of the field in the data payload returned by the Preprocess flow to use for a flow Name. */
   flowNameField: string;
-  /** Optional name of the field in the data payload returned by the Preprocess Flow to use for an External Customer Id. */
+  /** Optional name of the field in the data payload returned by the Preprocess flow to use for an External Customer Id. */
   externalCustomerIdField?: string;
-  /** Optional name of the field in the data payload returned by the Preprocess Flow to use for an External Customer User Id. */
+  /** Optional name of the field in the data payload returned by the Preprocess flow to use for an External Customer User Id. */
```

Comment:
nit: Since TS indicates if this field is optional, wondering if `Optional` is necessary.

Reply:
Good point; I'll do a sweep of all instances of `/** Optional...`